docs/plugins/pluggable_rule_engine.md

```diff
@@ -11,7 +11,6 @@ The iRODS Consortium currently supports the following rule engine plugins:
  - [C++ Default Policy Rule Engine Plugin](https://github.com/irods/irods/blob/main/plugins/rule_engines/src/cpp_default_policy.cpp)
  - [C++ Audit (AMQP) Rule Engine Plugin](https://github.com/irods/irods_rule_engine_plugin_audit_amqp)
  - [C++ Logical Quotas Rule Engine Plugin](https://github.com/irods/irods_rule_engine_plugin_logical_quotas)
- - [C++ Metadata Guard Rule Engine Plugin](https://github.com/irods/irods_rule_engine_plugin_metadata_guard)
```

Comment:
Could make this point to the new section.

That would allow it to be listed with the other plugins.

Reply:
can we make it the same as the C++ default policy plugin three above?

logical quotas will have the same treatment at some point ...

Reply:
That points to the code though. How would someone find the documentation for the plugin? Where would that live?

That also raises the question of - where are the docs for the C++ default REP? As I understand it, we don't provide any documentation for it right now.

Reply:
ah, i see.   correct on both points.

Reply:
Created https://github.com/irods/irods/issues/8290 for documenting the C++ default REP.

Reply:
Linked to the new section.